litellm/llms/databricks/chat.py

```diff
@@ -635,6 +645,7 @@ def embedding(
             self.client = client
 
         ## EMBEDDING CALL
+        response: httpx.Response = None
```

Comment:
Caught by pyright: `response` was undefined in the `except` block if an exception was thrown on line 650